spec.html

```diff
@@ -4055,17 +4045,20 @@ <h1>CopyDataBlockBytes ( _toBlock_, _toIndex_, _fromBlock_, _fromIndex_, _count_
       </emu-clause>
     </emu-clause>
 
-    <emu-clause id="sec-privatefield-record-specification-type">
-      <h1>The PrivateField Record Specification Type</h1>
-      <p>The PrivateField type is a Record used in the specification of private class fields.</p>
-      <p>Values of the PrivateField type are Record values whose fields are defined as by <emu-xref href="#table-privatefield-fields"></emu-xref>. Such values are referred to as <dfn>PrivateField Record</dfn>s.</p>
-      <emu-table id="table-privatefield-fields" caption="PrivateField Record Fields">
+    <emu-clause id="sec-privateelement-record-specification-type">
+      <h1>The PrivateElement Record Specification Type</h1>
+      <p>The PrivateElement type is a Record used in the specification of private class fields, methods, and accessors.</p>
+      <p>Values of the PrivateElement type are Record values whose fields are defined as by <emu-xref href="#table-privateelement-fields"></emu-xref>. Such values are referred to as <dfn>PrivateElement Record</dfn>s.</p>
+      <emu-table id="table-privateelement-fields" caption="PrivateElement Record Fields">
         <table>
           <tbody>
           <tr>
             <th>
               Field Name
             </th>
+            <th>
+              Values of the [[Kind]] field for which it is present
```

Comment:
Do we have other cases besides Property Descriptors where a Record's shape isn't consistent? Is there a strong reason to make the presence of slots conditional, rather than have them present but `~empty~` when they aren't relevant? (i realize this was already present in the base PR)

Reply:
> Do we have other cases besides Property Descriptors where a Record's shape isn't consistent?

Eh, kind of? Environment Records have a sort of type heirarchy, where the subtypes introduce additional fields, which is like this. Nothing else I can think of offhand.

I like the symmetry between this type and Property Descriptor, but I'm happy either way. I wouldn't want to include that change in this meta-PR, though.